src/components/shared/Button/Button.styles.js

```diff
@@ -0,0 +1,8 @@
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+
+export const DefaultButtonStyle = styled(Button)(({}) => ({
+  padding: '0.5rem 1rem',
```

Comment:
The name `DefaultButtonStyle` sounds more like a plain style object ( and we are also working with the convention that when we will have a shared component we can add a style object so this is confusing). 

Please change this into more meaningful name